database/dynamodb.go

```diff
@@ -68,3 +83,37 @@ func CreateDynamoDB() *dynamodb.DynamoDB {
 	db = dynamodb.New(sess)
 	return db
 }
+
+func ProcessGetFeatureFlagByHashKey(attributeName string, attributeValue string) (*models.FeatureFlag, error) {
+
+	db := CreateDynamoDB()
+
+	input := &dynamodb.GetItemInput{
+		TableName: aws.String(GetFeatureFlagTableName()),
+		Key: map[string]*dynamodb.AttributeValue{
+			attributeName: { 
+				S: aws.String(attributeValue),
+			},
+		},
+	}
+
+	result, err := db.GetItem(input)
+
+	if err != nil {
+		utils.DdbError(err)
+		return nil, err
+	}
+
+	if len(result.Item) == 0{
+		return nil, nil
+	}
+
+	featureFlag := new(models.FeatureFlag)
```

Comment:
This line is creating a new models.FeatureFlag struct. However, the struct is not being initialized. This means that the struct will have all of its fields set to the zero value. For example, the Name field will be set to the empty string and the Enabled field will be set to false.

Reply:
Same comment as above, so resolving this @iamitprakash